### PR TITLE
CAPP Twaalf Landing Page 🇳🇱

### LINKED ISSUE #1149: CAPP12 - first landing page
Maybe based on the V2 version I made for CAPP Agile Learning in this abandoned PR:  #755 

- [ ] Landing page (which URL?)
- [ ] Announcement box Home (or pop-up)
- [ ] Announcement box CAPP LMS (or pop-up)

First 🇳🇱 only

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
### Todo: 

- [x] Add page
- [x] Hero
- [x] Content
- [x] Conversion / Form
- [x] Announce
- [x] Screenshots

Close #1149

### CODE REVIEW

File: source/stylesheets/_capp-new-version.scss [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    }
+  }
+
+  .hero-inner p {
+    margin-bottom: 0;
+  }
+
+  .capp-introduction {
+    $background-image: "../images/capp-12-intro-bg.svg";
+    background-color: #353741;
```
[the remark this answers is not in the document]
Comment (RonaldDijkstra) [reply source/stylesheets/_capp-new-version.scss]: Ah man!

**Comment by RonaldDijkstra** [comment]
Screenshot van huidige status: 

landingpage-capp-zwoelf-0

**Comment by RonaldDijkstra** [comment]
Deployed to staging, which still isn't working on our own dns (use 1.1.1.1)

https://staging.defacto.nl/capp-learning-talent-management-system/

**Comment by RonaldDijkstra** [comment]
Used text is from the (concept) productsheet. It's just a better placeholder than http://slipsum.com/, no more no less.

**Comment by sn3p** [comment]
What kind of CTA/conversion do we want on the page? Contact? Productsheet download?

File: locales/nl.yml [hunk 2]
```diff
@@ -117,6 +117,9 @@ nl:
       img: screenshots/capp-agile-demo_nl.png
       img_alt: "Vraag een demo aan"
     testimonials_heading: "Wat onze klanten zeggen"
+  capp_new:
+    heading_contact: "Nieuwsgierig? Laat uw gegevens achter dan nemen we contact op."
+    formkeep_title: "Afspraak maken CAPP 12"
```
Comment (fatboypunk) [inline locales/nl.yml]: Shouldn't this be just CAPP btw?

Comment (RonaldDijkstra) [reply locales/nl.yml]: I need to specify for sales for which version a request is, so changed it to 'Afspraak maken nieuwe CAPP'

**Comment by RonaldDijkstra** [comment]
@sn3p we do not have a downloadable productsheet yet, so just a form will do for now

**Pull request merged**